modules/bmc/redfish.rb

```
@@ -0,0 +1,210 @@
+require 'redfish_client'
+require 'bmc/base'
+require 'bmc/redfish/dell'
+require 'bmc/redfish/hpe'
+
+module Proxy
+  module BMC
+    class Redfish < Base
+      include Proxy::Log
+      include Proxy::Util
+
+      def initialize(args)
+        @redfish_verify_ssl = Proxy::BMC::Plugin.settings.bmc_redfish_verify_ssl
+        # enforce SSL verification as a default, if the value isn't set in the config
+        # file, or is set to a "garbage" value (anything but boolean false)
+        @redfish_verify_ssl = true unless @redfish_verify_ssl == false
```

This feels odd, if you really want true/false here, wouldn't it be better to crash hard rather than silently ignore a "wrong" value and use a different one instead?